### ISSUE TITLE
Latest Backstage release (release-2021-05-27) breaks this plugin

### ISSUE THREAD

**Issue by ConnorDY** [RoadieHQ/backstage-plugin-github-pull-requests#102]
When trying to navigate to the Pull Requests tab, the page crashes with the following error/stacktrace:
```
Uncaught TypeError: Cannot read property 'metadata' of undefined
    at isGithubPullRequestsAvailable (index.esm.js?b593:396)
    at Router (index.esm.js?b593:400)
    at renderWithHooks (react-dom.development.js?9f10:14804)
    at mountIndeterminateComponent (react-dom.development.js?9f10:17483)
    at beginWork (react-dom.development.js?9f10:18597)
    at HTMLUnknownElement.callCallback (react-dom.development.js?9f10:189)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?9f10:238)
    at invokeGuardedCallback (react-dom.development.js?9f10:293)
    at beginWork$1 (react-dom.development.js?9f10:23204)
    at performUnitOfWork (react-dom.development.js?9f10:22158)
react-dom.development.js?9f10:19528 The above error occurred in the <Router> component:
    in Router (created by RoutableExtension(Router))
    in RoutableExtension(Router) (created by Extension(Component))
    in Suspense (created by Extension(Component))
    in Extension(Component) (at EntityPage.tsx:124)
    in article (created by Content)
    in Content (created by RoutedTabs)
    in RoutedTabs (created by EntityLayout)
    in div (created by Page)
    in ThemeProvider (created by Page)
    in Page (created by EntityLayout)
    in EntityLayout (at EntityPage.tsx:118)
    in EntitySwitch (at EntityPage.tsx:154)
    in EntitySwitch (at EntityPage.tsx:272)
    in Outlet (created by CatalogEntityPage)
    in EntityLoaderProvider (created by CatalogEntityPage)
    in CatalogEntityPage (created by RoutableExtension(CatalogEntityPage))
    in RoutableExtension(CatalogEntityPage) (created by Extension(Component))
    in Suspense (created by Extension(Component))
    in Extension(Component) (at App.tsx:72)
    in Route (at App.tsx:72)
    in FlatRoutes (at App.tsx:69)
    in div (created by SidebarPage)
    in SidebarPage (at App.tsx:67)
    in Route (created by AppRouter)
    in Routes (created by AppRouter)
    in SignInPageWrapper (created by AppRouter)
    in Router (created by BrowserRouter)
    in BrowserRouter (created by AppRouter)
    in AppRouter (at App.tsx:66)
    in RoutingProvider (created by Provider)
    in CssBaseline (created by WithStyles(CssBaseline))
    in WithStyles(CssBaseline) (created by AppThemeProvider)
    in ThemeProvider (created by AppThemeProvider)
    in AppThemeProvider (created by Provider)
    in AppContextProvider (created by Provider)
    in ApiProvider (created by Provider)
    in Provider (at App.tsx:63)
    in App (at src/index.tsx:6)
Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
logCapturedError @ react-dom.development.js?9f10:19528
scheduler.development.js?2f5b:178 Uncaught TypeError: Cannot read property 'metadata' of undefined
    at isGithubPullRequestsAvailable (index.esm.js?b593:396)
    at Router (index.esm.js?b593:400)
    at renderWithHooks (react-dom.development.js?9f10:14804)
    at mountIndeterminateComponent (react-dom.development.js?9f10:17483)
    at beginWork (react-dom.development.js?9f10:18597)
    at HTMLUnknownElement.callCallback (react-dom.development.js?9f10:189)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?9f10:238)
    at invokeGuardedCallback (react-dom.development.js?9f10:293)
    at beginWork$1 (react-dom.development.js?9f10:23204)
    at performUnitOfWork (react-dom.development.js?9f10:22158)
```

## Expected Behavior

Plugin should work as expected and show PRs.

## Current Behavior

Plugin displays a white page.

## Steps to Reproduce

Use this plugin with the latest Backstage release and navigate to the Pull Requests tab.

## Context

Latest backstage release:
https://github.com/backstage/backstage/releases/tag/release-2021-05-27

## Your Environment

N/A

**Comment by Irma12** [comment]
Hey @ConnorDY thank you for reporting this issue, we will take a look and hopefully resolve it soon.

**Comment by Irma12** [comment]
Hi @ConnorDY we have been able to identify the issue and resolved it in new version (v1.0.5). Please let us know if you still experience issues.

**Comment by grv-trip** [comment]
Was running into the same issue, upgrading the plugin to 1.0.5 as mentioned by @Irma12 worked.
